deepchecks/vision/checks/distribution/image_dataset_drift.py

```diff
@@ -38,9 +38,10 @@ class ImageDatasetDrift(TrainTestCheck):
 
     Parameters
     ----------
-    alternative_image_properties : List[str] , default: None
-        List of alternative image properties names. Must be attributes of the ImageFormatter classes that are passed to
-        train and test's VisionData class. If None, check uses DEFAULT_IMAGE_PROPERTIES.
+    alternative_image_properties : Dict[str, Callable] , default: None
+        Dict of image properties. Replaces the default deepchecks properties.
+        Each property have a name (key of dict) and function which accepts a `List[np.ndarray]` as input. If None, check
```

Comment:
```suggestion
        Each property has a name (key of dict) and function which accepts a `List[np.ndarray]` as input. If None, check
```

Reply:
Also, phrasing unclear. Would say that "each dictionary is comprised of name (key) and function (value)" or something of that sort